binrw/doc/attribute.md

```diff
@@ -858,22 +858,36 @@ assert_eq!(output.into_inner(), b"\0\0\0\x04\0\0\0\x01");
 ```
 </div>
 
-<div class="br">
-
 # Conditional values
 
-The `if` directive allows conditional parsing of a field, reading from data
-if the condition is true and using a computed value if the condition is
+The `if` directive allows conditional
+<span class="br">parsing</span><span class="bw">serialisation</span> of a field,
+<span class="br">reading</span><span class="bw">writing</span> data if the
+condition is true and optionally using a computed value if the condition is
 false:
 
+<div class="br">
+
 ```text
 #[br(if = $cond:expr)] or #[br(if($cond:expr))]
 #[br(if = $cond:expr, $alternate:expr)] or #[br(if($cond:expr, $alternate:expr))]
 ```
+</div>
+<div class="bw">
+
+```text
+#[bw(if = $cond:expr)] or #[bw(if($cond:expr))]
+#[bw(if = $cond:expr, $alternate:expr)] or #[bw(if($cond:expr, $alternate:expr))]
```

Comment:
wait no

Reply:
I feel like you are letting your personal feelings over the accuracy of the documentation get in the way of approving this pull request, so I fixed the documentation.